css/styles.css

```diff
@@ -33,19 +33,19 @@ h2 {
   position: relative;
 }
 
-.div_question-card {
+.question-card__button {
   display: flex;
   flex-direction: column;
   align-items: center;
 }
 
-.question-card_categories {
+.question-card__categories {
   list-style-type: none;
   display: flex;
   flex-direction: row;
```

Comment:
```suggestion
```
;) You never have to declare `row`. It's the default.

Reply:
sure